static/app/utils/useRelease.tsx

```diff
@@ -26,11 +13,15 @@ export function useRelease({
   releaseVersion: string;
   enabled?: boolean;
 }) {
-  return useApiQuery<Release>(
-    getReleaseQueryKey({orgSlug, projectSlug, releaseVersion}),
-    {
-      enabled,
+  return useQuery({
+    ...apiOptions('/projects/$orgSlug/$projectSlug/releases/$releaseVersion/', {
```

Comment:
wouldn't really need to spread here right

Reply:
we need the spread because additionally, `enabled` gets passed. `apiOptions` does _not_ accept arbitrary `QueryOptions` because making those generics is quite hard, especially with `select`. Instead, it creates “base” options you’d want to share with all usages and then you can apply more selectively to each call-side with object spread.